sql/handler.cc

```diff
@@ -7095,10 +7095,10 @@ bool Vers_parse_info::fix_alter_info(THD *thd, Alter_info *alter_info,
     // copy info from existing table
```

Comment:
this is intended to be a fixup for `CREATE` commit, right?

Reply:
What exactly *this*? The thread doesn't show any code.

Reply:
I mean, the whole commit. Github just doesn't allow leave a comment for a commit, not for line

Reply:
I don't see info on commit neither. You can put commit hash/message so I could understand of what are we talking about.

Reply:
If it's about "Superfluous TABLE_SHARE in period_info_t": yes, this is for `CREATE` code.